restful_resource.gemspec

```diff
@@ -27,11 +27,11 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'rspec_junit_formatter'
 
   spec.add_dependency 'activesupport', '>= 6', '< 8'
-  spec.add_dependency 'faraday', '~> 1.10'
+  spec.add_dependency 'faraday', '>= 1.10'
   spec.add_dependency 'faraday-cdn-metrics', '~> 0.2'
   spec.add_dependency 'faraday-encoding'
   spec.add_dependency 'faraday-http-cache', '~> 2.2'
   spec.add_dependency 'faraday_middleware', '~> 1.0'
   spec.add_dependency 'link_header'
-  spec.add_dependency 'rack', '~> 2.2'
+  spec.add_dependency 'rack', '>= 3.1'
```

Comment:
We need to update this as `sinatra >= 4.0.0` depends on `rack >= 3.0.0, < 4`, and there is a [venerability in Sinatra](https://github.com/carwow/flatmin/security/dependabot/156) in versions `< 4.1.0`